tests/test_probes.py

```diff
@@ -0,0 +1,8 @@
+from fastapi import FastAPI, Request
+
+app = FastAPI(liveness_probe_url="/health/liveness")
```

Comment:
You probably need to add a test for readiness, since this PR is making code coverage drop from 100%.